addons/jr/jr_classes.hpp

```diff
@@ -76,6 +77,20 @@ class asdg_OpticRail1913: asdg_OpticRail { // the "medium" rail, long enough to
         optic_KHS_hex = 1;
         optic_KHS_old = 1;
         optic_KHS_tan = 1;
+		//dpapexaddon

```

Comment:
the 4 spaces for indent does not fit the files current structure as I see it.... and if you guys want to use it please do and make what ever changes you need. If spaces and indents are what you are worried about when a free lance community guys helps update then im helping the wrong people.